core/storage/exploration/gae_models.py

```diff
@@ -333,7 +333,7 @@ def compute_models_to_commit(
         )
 
         # Here we use cast because the additional_models is list of BaseModels
-        # and we want to hint the MyPy that this is ExplorationRightsModel.
+        # and we want to hint to MyPy that this is ExplorationRightsModel.
```

Comment:
Why can't we handle this in the typedefs at the top of the method?

Reply:
Fixed